libs/core/src/lib/icon-tab-bar/components/icon-tab-bar-filter-type/icon-tab-bar-filter-type.component.html

```diff
@@ -0,0 +1,60 @@
+<ul
+    class="fd-icon-tab-bar__header"
+    role="tablist"
+    fdOverflowItems
+    [isRtl]="isRtl"
+    itemSelector="[data-overflowItem]"
+    [overflowOffset]="100"
+    (overflowChanged)="_recalculateVisibleItems($event)"
+>
+    <li
+        *ngIf="showTabAll"
+        class="fd-icon-tab-bar__item"
+        role="presentation"
+        data-overflowItem
+    >
+        <a (click)="_selectItem(_tabs[0])"
+           [attr.aria-selected]="_tabs[0]?.uId === _selectedUid"
+           class="fd-icon-tab-bar__tab"
+           role="tab"
+        >
+            <div class="fd-icon-tab-bar__container fd-icon-tab-bar__container--filter">
+                <span *ngIf="tabsConfig[0]?.counter" class="fd-icon-tab-bar__filter-counter">{{tabsConfig[0]?.counter}}</span>
+                <span class="fd-icon-tab-bar__filter-label">{{tabsConfig[0]?.label}}</span>
+            </div>
+        </a>
+    </li>
+    <li *ngFor="let item of _tabs | slice:1"
```

Comment:
I would prefer splitting tab items in typescript into 2 variables for first tab in an array, and the rest

Reply:
@N1XUS @platon-rov I created variable for main tab, but could create second var for array, because it would be tricky. This will complicate the logic and understanding of the code. Every action I so something with main array (change styles, tabs orders etc). To support second array I need wrap every parent method. 

I don't see anything wrong with the `slice` pipe. It is cached by the angular and is called only when the reference to the array has changed, this is extremely rare and the array itself cannot be large.

We can also wrap all the tabs in a ng-container wrapper and hang a *ngFor on it, and inside check and render the template of the main tab or the common one. But it will complicate the code too.